napari/_qt/qt_layerlist.py

```diff
@@ -285,7 +284,7 @@ def dropEvent(self, event):
         total = self.vbox_layout.count() // 2 - 1
         insert = total - divider_index
         index = self.layers.index(self.drag_name)
-        layer = self.layers[index]
+        # layer = self.layers[index]
```

Comment:
i take it this isn't being used? feel free to delete instead of just comment out